models/intermediate/int_xero__tracking_categories_with_options.sql

```diff
@@ -0,0 +1,42 @@
+{{ config(
+    materialized='ephemeral',
```

Comment:
Based on other comments, we may need to change this to a table or view

Reply:
Removed this materialization.